### DISCUSSION TITLE
Core reload config from db after receive an unix signal

### DISCUSSION THREAD

**Discussion by mywalkb** [LSPosed/LSPosed#1019] [Ideas]
Is it possible implement in core process a reload config from db when it receive an unix signal?
For example signal HUP used by many unix project for reload config.
I change db config externally and send the signal HUP to lspd and it reload config from db.
Is it too much complex?

**Comment by yujincheng08** [comment]
The core daemon process is mainly in java so there's no way for us to integrate a signal handler.
